stetho/src/main/java/com/facebook/stetho/common/ListAccumulator.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.common;
+
+import java.util.List;
+
+public interface ListAccumulator<E> extends List<E>, Accumulator<E> {
```

Comment:
Well, for one, because standard convention is to do stuff like `List<T> list = new ArrayList<>()` ? I was also experimenting with other ways to wrap both List and Accumulator, but I just happened to only end up utilizing ArrayListAccumulator.

I wouldn't say the diff is "way overrotating on generalization" ... I'm trying to strike a balance here. Sometimes having some generalization might cost 1 allocation and add 1 Class<?>, but save a ton of byte code elsewhere by removing the need for repeated code, which also has the benefit of improving maintainability.